src/EventGridEmulator/Program.cs

```diff
@@ -22,7 +24,15 @@
 
 builder.Services.AddOptions<TopicOptions>().BindConfiguration(string.Empty);
 builder.Services.AddSingleton<SubscriberPolicyHttpMessageHandler>();
-builder.Services.AddHttpClient(SubscriberConstants.HttpClientName, SubscriberConstants.ConfigureHttpClient).AddHttpMessageHandler<SubscriberPolicyHttpMessageHandler>();
+builder.Services.AddHttpClient(SubscriberConstants.HttpClientName, SubscriberConstants.ConfigureHttpClient)
+    .AddHttpMessageHandler<SubscriberPolicyHttpMessageHandler>()
+    .ConfigurePrimaryHttpMessageHandler(handler => new HttpClientHandler
+    {
+        ClientCertificateOptions = ClientCertificateOption.Manual,
```

Comment:
I wonder if the four lines are required? If not, can we find and keep only those that are actually needed?